<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>main</class>
    <widget class="QMainWindow" name="main">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>380</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="maximumSize">
            <size>
                <width>600</width>
                <height>380</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>PlagDef</string>
        </property>
        <property name="windowIcon">
            <iconset resource="resources.qrc">
                <normaloff>:/res/icon.ico</normaloff>:/res/icon.ico
            </iconset>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(77, 80, 87);</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="minimumSize">
                <size>
                    <width>600</width>
                    <height>380</height>
                </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="sizeConstraint">
                    <enum>QLayout::SetMaximumSize</enum>
                </property>
                <property name="leftMargin">
                    <number>20</number>
                </property>
                <property name="topMargin">
                    <number>10</number>
                </property>
                <property name="rightMargin">
                    <number>20</number>
                </property>
                <property name="bottomMargin">
                    <number>20</number>
                </property>
                <item>
                    <spacer name="top_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                            <enum>QSizePolicy::Expanding</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>10</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QLabel" name="logo_label">
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>45</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/res/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <spacer name="logo_process_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                            <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>30</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <layout class="QGridLayout" name="process_actions_grid">
                        <item row="0" column="0">
                            <layout class="QVBoxLayout" name="docs_layout">
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetDefaultConstraint</enum>
                                </property>
                                <item>
                                    <widget class="QLabel" name="docs_label">
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                                                style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Documents&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="docs_icon">
                                        <property name="minimumSize">
                                            <size>
                                                <width>130</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img
                                                src=&quot;:/res/add_circle_white.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="0" column="1" colspan="2">
                            <widget class="Line" name="docs_lang_line">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>50</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="lineWidth">
                                    <number>0</number>
                                </property>
                                <property name="midLineWidth">
                                    <number>5</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <layout class="QVBoxLayout" name="lang_layout">
                                <item>
                                    <widget class="QLabel" name="lang_label">
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                                                style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Language&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="lang_icon">
                                        <property name="minimumSize">
                                            <size>
                                                <width>130</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img
                                                src=&quot;:/res/lang_white.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="0" column="4">
                            <widget class="Line" name="lang_detect_line">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>50</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <property name="lineWidth">
                                    <number>0</number>
                                </property>
                                <property name="midLineWidth">
                                    <number>5</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="5">
                            <layout class="QVBoxLayout" name="detect_layout">
                                <item>
                                    <widget class="QLabel" name="detect_label">
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                                                style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Detection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="detect_icon">
                                        <property name="minimumSize">
                                            <size>
                                                <width>130</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img
                                                src=&quot;:/res/plagiarism_white.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                            </string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="1" column="0" colspan="2">
                            <layout class="QVBoxLayout" name="open_layout">
                                <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                                    <widget class="QPushButton" name="open_button">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>130</width>
                                                <height>50</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">QPushButton {
                                                border-radius: 25px;
                                                border: 3px solid rgb(242, 141, 1);
                                                border-style: outset;
                                                color: rgb(255, 255, 255);
                                                }

                                                :hover{
                                                background-color: rgb(242, 141, 1);
                                                }

                                                :pressed {
                                                border-style: inset;
                                                }
                                            </string>
                                        </property>
                                        <property name="text">
                                            <string>Open folder</string>
                                        </property>
                                        <property name="icon">
                                            <iconset resource="resources.qrc">
                                                <normaloff>:/res/folder_open_white.png</normaloff>
                                                :/res/folder_open_white.png
                                            </iconset>
                                        </property>
                                        <property name="autoDefault">
                                            <bool>false</bool>
                                        </property>
                                        <property name="default">
                                            <bool>false</bool>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="1" column="2">
                            <spacer name="open_lang_spacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Expanding</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item row="1" column="3">
                            <layout class="QVBoxLayout" name="verticalLayout">
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetDefaultConstraint</enum>
                                </property>
                                <item alignment="Qt::AlignHCenter">
                                    <widget class="QRadioButton" name="eng_radio">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>100</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">color: rgb(255, 255, 255);</string>
                                        </property>
                                        <property name="text">
                                            <string>English</string>
                                        </property>
                                        <property name="icon">
                                            <iconset resource="resources.qrc">
                                                <normaloff>:/res/uk_flag.png</normaloff>:/res/uk_flag.png
                                            </iconset>
                                        </property>
                                        <property name="iconSize">
                                            <size>
                                                <width>32</width>
                                                <height>32</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item alignment="Qt::AlignHCenter">
                                    <widget class="QRadioButton" name="ger_radio">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>100</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">color: rgb(255, 255, 255);</string>
                                        </property>
                                        <property name="text">
                                            <string>German</string>
                                        </property>
                                        <property name="icon">
                                            <iconset resource="resources.qrc">
                                                <normaloff>:/res/ger_flag.png</normaloff>:/res/ger_flag.png
                                            </iconset>
                                        </property>
                                        <property name="iconSize">
                                            <size>
                                                <width>32</width>
                                                <height>32</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="1" column="4">
                            <spacer name="lang_detect_spacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeType">
                                    <enum>QSizePolicy::Preferred</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item row="1" column="5">
                            <layout class="QVBoxLayout" name="start_detect_layout">
                                <item alignment="Qt::AlignHCenter">
                                    <widget class="QPushButton" name="detect_button">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>128</width>
                                                <height>50</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">QPushButton {
                                                border-radius: 25px;
                                                border: 3px solid rgb(242, 141, 1);
                                                border-style: outset;
                                                color: rgb(255, 255, 255);
                                                }

                                                :hover{
                                                background-color: rgb(242, 141, 1);
                                                }

                                                :pressed {
                                                border-style: inset;
                                                }
                                            </string>
                                        </property>
                                        <property name="text">
                                            <string>Find matches</string>
                                        </property>
                                        <property name="icon">
                                            <iconset resource="resources.qrc">
                                                <normaloff>:/res/search_white.png</normaloff>:/res/search_white.png
                                            </iconset>
                                        </property>
                                        <property name="autoDefault">
                                            <bool>false</bool>
                                        </property>
                                        <property name="default">
                                            <bool>false</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QCheckBox" name="xml_radio">
                                        <property name="styleSheet">
                                            <string notr="true">color: rgb(255, 255, 255);</string>
                                        </property>
                                        <property name="text">
                                            <string>Generate XML reports</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
                <item>
                    <spacer name="bottom_spacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeType">
                            <enum>QSizePolicy::Expanding</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>32</height>
                            </size>
                        </property>
                    </spacer>
                </item>
            </layout>
        </widget>
    </widget>
    <resources>
        <include location="resources.qrc"/>
    </resources>
    <connections/>
</ui>
